lib/extensions/conditional/active_record/base.rb

```diff
@@ -20,4 +22,12 @@ def specific_conditions
       conditions.map(&:actable)
     end
   end
+
+  module ConditionInstanceMethods
+    # A human-readable name for each condition; usually just wraps a title
+    # or name field. Meant to be used in a polymorphic manner for views.
+    def name
+      fail 'To be implemented by each condition.'
```

Comment:
How do we test this line? All models which declare `acts_as_conditional` and override this method are covered. Do we include this module in a dummy object?